tmt/steps/provision/bootc.py

```diff
@@ -410,6 +429,10 @@ def go(self, *, logger: Optional[tmt.log.Logger] = None) -> None:
         built_image.rename(renamed_image)
         data.image = f"file://{renamed_image}"
 
+        if data.build_disk_image_only:
+            self.info(f"The disk image is saved at: {data.image}")
```

Comment:
Ha, maybe we should use `self.print()` instead - `info` & co. are rather logging, progress-or-event reporting set of functions, but this piece of information looks like the actual output of the plugin, similar to `tmt * export` or `minute --print-flavors`.

I would also drop the "The disk image ..." part, to make things easier for machine consumers of the output.

Reply:
Updated. Please review.